src/query/sql/src/planner/binder/aggregate.rs

```diff
@@ -55,6 +57,9 @@ pub struct AggregateInfo {
     /// Arguments of aggregation functions
     pub aggregate_arguments: Vec<ScalarItem>,
 
+    /// infos for window with aggregate function
+    pub window_info: Option<WindowInfo>,
```

Comment:
There will be multiple windows if there are window functions with different partition definitions? For example:
```sql
SELECT count(*) over(partition by a), count(*) over(partition by b) FROM t
```